firmware/app/scripts/astyle_format_west_command.py

```
@@ -0,0 +1,61 @@
+
+        if(args.dry_run):
+            astyle_args.append("--dry-run")
+        astyle_args.extend(
+            [
+                "--options={}/../app/misc/astyle.cfg".format(os.environ["ZEPHYR_BASE"]),
+                "--recursive",
+                "{}/../app/src/*.c,*.h".format(os.environ["ZEPHYR_BASE"]),
+            ]
+        )
+
+        with subprocess.Popen(
```

Ruff (0.14.0)

`subprocess` call: check for execution of untrusted input

(S603)